### PR TITLE
September 2023 releases

### PR DESCRIPTION
**Pull request by kenodegard**
### Description

This is a large announcement so any help to review is much appreciated.

### CODE REVIEW

File: blog/2023-07-05-conda-libmamba-solver-rollout.mdx [hunk 1]
```diff
@@ -10,6 +10,13 @@ tags: [conda-libmamba-solver, solvers, performance]
 
 Over a year ago, the conda team at Anaconda, together with conda developers at Quansight and QuantStack, [announced the release](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) of a new solver for conda based on the libraries that the [mamba project](https://mamba.readthedocs.io/) had developed on top of the venerable libsolv project.
 
+:::info Update
+
+The default solver change for conda will be occurring in a __special 23.10.0 release__ (not in 23.9.0). See the special announcement [here](https://github.com/conda/conda/blob/main/CHANGELOG.md#2390-2023-09-27)
```
Comment (kenodegard) [inline blog/2023-07-05-conda-libmamba-solver-rollout.mdx:15]:
```suggestion
The default solver change for conda will be occurring in a __special 23.10.0 release__ (not in 23.9.0). See the special announcement [here](https://github.com/conda/conda/blob/main/CHANGELOG.md#2390-2023-09-27).
```

File: blog/2023-10-12-september-releases.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+## Changes in Conda [23.9.0](https://github.com/conda/conda/releases/tag/23.9.0)
+
+To update conda, run:
+```bash
+conda install -n base conda=23.9.0
+```
+
+### 📢 Special Announcement 📢
+
+> Conda's default solver will change to [`conda-libmamba-solver`](https://conda.github.io/conda-libmamba-solver/) in a __special 23.10.0 release__ in the near future!
```
Comment (kenodegard) [inline blog/2023-10-12-september-releases.md:21]:
```suggestion
:::info

Conda's default solver will change to [`conda-libmamba-solver`](https://conda.github.io/conda-libmamba-solver/) in a __special 23.10.0 release__ in the near future!

:::
```

File: blog/2023-07-05-conda-libmamba-solver-rollout.mdx [hunk 3]
```diff
@@ -10,6 +10,13 @@ tags: [conda-libmamba-solver, solvers, performance]
 
 Over a year ago, the conda team at Anaconda, together with conda developers at Quansight and QuantStack, [announced the release](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) of a new solver for conda based on the libraries that the [mamba project](https://mamba.readthedocs.io/) had developed on top of the venerable libsolv project.
 
+:::info Update
```
Comment (kenodegard) [inline blog/2023-07-05-conda-libmamba-solver-rollout.mdx:13]:
```suggestion
:::caution Update
```

File: blog/2023-10-12-september-releases.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+To update conda-index, run:
+```bash
+conda install -n base pycosat=0.6.6
+```
+
+### ✨ What's New? ✨
+
+* Pycosat 0.6.4 accidentally did not include the changes intended to be released. Pycosat 0.6.5 includes those changes.
+* Use `PyMem_Calloc()` to initialize memory to `0`.
```
Comment (beeankha) [inline blog/2023-10-12-september-releases.md:173]:
```suggestion
* `PyMem_Calloc()` is utilized in order to initialize memory to `0`.
```

File: blog/2023-10-12-september-releases.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+:::info
+
+Conda's default solver will change to [`conda-libmamba-solver`](https://conda.github.io/conda-libmamba-solver/) in a __special 23.10.0 release__ in the near future!
+
+:::
+
+You can already benefit from it _today_ by [configuring your conda installation to use it](https://conda.github.io/conda-libmamba-solver/getting-started/#usage) (e.g. by running `conda config --set solver libmamba`).
+
+The current "classic" solver is based on [pycosat](https://github.com/conda/pycosat)/[Picosat](http://fmv.jku.at/picosat/) and will remain part of conda for the forseeable future, a fallback is possible and available.
```
Comment (tnabtaf) [inline blog/2023-10-12-september-releases.md:29]:
```suggestion
The current "classic" solver is based on [pycosat](https://github.com/conda/pycosat)/[Picosat](http://fmv.jku.at/picosat/) and will remain part of conda for the foreseeable future. A fallback is possible and available.
```

**Review by tnabtaf — APPROVED** [review]
Ken, this looks good to me.  I have one minor suggestion - see the review.

Also, I think you'll want to run `bin/update-news` before publishing.  That will add this post to the list on the home page

File: blog/2023-10-12-september-releases.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+* * *
+
+### We ❤️ Our Community
+
+Altogether we had 10 (!) new contributors this release cycle; thank you to all of our open source community members for helping making these improvements possible.
+
+* [@boldorider4](https://github.com/boldorider4) made their first contribution in [conda-build#4960](https://github.com/conda/conda-build/pull/4960)
+* [@DaveKaretnyk](https://github.com/DaveKaretnyk) made their first contribution in [conda-build#5004](https://github.com/conda/conda-build/pull/5004)
+* [@jmcarpenter2](https://github.com/jmcarpenter2) made their first contribution in [conda#13034](https://github.com/conda/conda/pull/13034)
```
Comment (beeankha) [inline blog/2023-10-12-september-releases.md:183]:
```suggestion
* [@DaveKaretnyk](https://github.com/DaveKaretnyk) made their first contribution in [conda-build#5004](https://github.com/conda/conda-build/pull/5004)
* [@dholth](https://github.com/dholth) made their first contribution in [pycosat#54](https://github.com/conda/pycosat/pull/54)
* [@jmcarpenter2](https://github.com/jmcarpenter2) made their first contribution in [conda#13034](https://github.com/conda/conda/pull/13034)
```

File: blog/2023-10-12-september-releases.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+### ✨ What's New? ✨
+
+* Pycosat 0.6.4 accidentally did not include the changes intended to be released. Pycosat 0.6.5 includes those changes.
+* Use `PyMem_Calloc()` to initialize memory to `0`.
+
+* * *
+
+### We ❤️ Our Community
+
+Altogether we had 10 (!) new contributors this release cycle; thank you to all of our open source community members for helping making these improvements possible.
```
Comment (beeankha) [inline blog/2023-10-12-september-releases.md:179]:
```suggestion
Altogether we had 11 (!) new contributors this release cycle; thank you to all of our open source community members for helping making these improvements possible.
```

File: blog/2023-10-12-september-releases.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kenodegard) [reply blog/2023-10-12-september-releases.md]: superseded by https://github.com/conda-incubator/conda-dot-org/pull/163#discussion_r1357212349

**Comment by beeankha** [comment]
Is there any way we can attach an image to this blog post? We could re-use one of the previous announcement images, either the 🎉 or 🚀 emoji image.

Currently the default image is a giant conda "C", and it also looks like the other blog posts' images aren't showing up on the home page for some reason:

Screenshot 2023-10-12 at 3 37 25 PM

File: blog/2023-10-12-september-releases.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+
+Additional details on this change as well as the full announcement can be found in the [conda 23.9.0 changelog](https://github.com/conda/conda/blob/main/CHANGELOG.md#2390-2023-09-27).
+
+<!-- truncate -->
+
+### ✨ What's New? ✨
+
+* A new "auth handler" plugin hook has been added for conda. See the new [`conda-auth`](https://github.com/conda-incubator/conda-auth) plugin for an example.
+* Index cache metadata is locked by default; a `--no-lock` option is also now available.
+* A new`context.register_envs` option can control whether to register environments in `~/.conda/environments.txt` when they are created. Defaults to true.
```
Comment (kathatherine) [inline blog/2023-10-12-september-releases.md:39]:
```suggestion
* A new `context.register_envs` option can control whether to register environments in `~/.conda/environments.txt` when they are created. Defaults to true.
```

File: blog/2023-10-12-september-releases.md [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+* The solver no longer bounces between two compatible solutions when the same command is run twice in a row.
+* Commands with no channels passed are handled gracefully.
+* Workaround added for missing `noarch` field in returned `PackageRecord` payload.
+* A bug where the `--prune` flag was not working correctly in `conda env update` commands has been fixed.
+* Environments are no longer aggressively updated to higher priority channels under certain conditions.
+* No more injection of channels from installed packages that do not exist or are unavailable.
+* All configured channels in `PackagesNotFoundError` exceptions print correctly.
+* No more crashes when a `MatchSpec` with a build string is specified in the CLI and there's a pinned spec for the same package name.
+* `defaults::pkg` workarounds are only applied for the default value `default_channels`.
+* Users won't be able to override pinned specs with incompatible CLI specs anymore. Instead they must modify their pinned specs explicitly.
```
Comment (kathatherine) [inline blog/2023-10-12-september-releases.md:140]:
```suggestion
* Users won't be able to override pinned specs with incompatible CLI specs anymore. Instead, they must modify their pinned specs explicitly.
```

File: blog/2023-10-12-september-releases.md [hunk 10]
```diff
@@ -0,0 +1,193 @@
...
+### ✨ What's New? ✨
+
+* Pycosat 0.6.4 accidentally did not include the changes intended to be released. Pycosat 0.6.5 includes those changes.
+* `PyMem_Calloc()` is utilized in order to initialize memory to `0`.
+
+* * *
+
+### We ❤️ Our Community
+
+Altogether we had 11 (!) new contributors this release cycle; thank you to all of our open source community members for helping making these improvements possible.
```
Comment (kathatherine) [inline blog/2023-10-12-september-releases.md:179]:
```suggestion
Altogether, we had 11 (!) new contributors this release cycle; thank you to all of our open source community members for helping making these improvements possible.
```

**Review by kathatherine — APPROVED** [review]
Yay, releases!

**Review by beeankha — APPROVED** [review]
I'm still seeing the large conda "C" logo vs the generic "🎉" image, but this isn't necessarily a blocker (especially with the work being done in [this PR](https://github.com/conda-incubator/conda-dot-org/pull/165))

**Pull request merged**